.github/workflows/tests.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           # Need to use `main` branch of optuna until https://github.com/optuna/optuna/pull/4589
           # is released in `optuna=3.2.0`. Currently only doing this when workflows are run
           # as that's the only place where the optuna + dask integration is being used.
-          python -m pip install git+https://github.com/optuna/optuna
+          python -m pip install git+https://github.com/optuna/optuna.git@378508ab6bddbad182bdfa0e8b3ad4bbb7040f00
```

Comment:
This is the commit where the fix referenced in the commit above was included. We could install directly from `main`, but pinning to this commit seemed favorable because it's less of a moving target.